packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -46,19 +75,40 @@ export default function NavigationMenuSelector( {
 					} ) }
 				/>
 			</MenuGroup>
-			{ showCreate && (
-				<MenuGroup>
-					<MenuItem onClick={ onCreateNew }>
-						{ __( 'Create new menu' ) }
-					</MenuItem>
-					<MenuItem
-						href={ addQueryArgs( 'edit.php', {
-							post_type: 'wp_navigation',
-						} ) }
-					>
-						{ __( 'Manage menus' ) }
-					</MenuItem>
-				</MenuGroup>
+			{ canUserCreateNavigation && (
+				<>
+					{ hasClassicMenus && (
+						<MenuGroup label={ __( 'Classic Menus' ) }>
+							{ classicMenus.map( ( menu ) => {
+								return (
+									<MenuItem
+										onClick={ () => {
+											convertClassicMenuToBlocks(
+												menu.id,
+												menu.name
+											);
+										} }
+										key={ menu.id }
+									>
+										{ decodeEntities( menu.name ) }
+									</MenuItem>
+								);
+							} ) }
+						</MenuGroup>
+					) }
+					<MenuGroup label={ __( 'Tools' ) }>
```

Comment:
What do you think about removing this label? I think _Create new menu_ and _Manage menus_ are clear enough to not need the additional context.

<img width="252" alt="Screen Shot 2022-01-24 at 15 17 38" src="https://user-images.githubusercontent.com/612155/150721315-54861439-3baf-4546-91a5-98bf203800f1.png">

Reply:
I think I'll leave it. To me it feels cleaner to me to consistently have headings when there are multiple groups like this.

If only there were designers in this timezone we could ask 😄

Reply:
I also like using different groups for Classic and Block menus.

Reply:
Cool, your call.

> If only there were designers in this timezone we could ask 😀

😅 yes unfortunately there wouldn't be enough time to do the backports if we wait. I put https://github.com/WordPress/gutenberg/pull/36311 into the 5.9.x project board so that we can iterate asap.